shared/src/model/exercise.js

```diff
@@ -87,7 +87,7 @@ class Exercise extends BaseModel {
 
   @computed get isMultiPart() { return this.questions.length > 1; }
   @computed get isSinglePart() { return this.questions.length == 1; }
-  @computed get isMultiChoice() { return every(this.questions, 'isMultiChoice'); }
+  @computed get isMultiChoice() { return every(this.questions, 'isMultipleChoice'); }
```

Comment:
I see that `isMultiChoice` and `isMultipleChoice` are being used, what is the difference?

Reply:
No difference, just kept it in sync the isMultiPart naming above. Hmm, maybe we should change them to be consistent. `isMultipleChoice` is the actual flag I need and I think it was a typo to call `isMultiChoice`, so this fixes that.